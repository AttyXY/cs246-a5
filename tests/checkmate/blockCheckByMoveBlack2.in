setup
+ k e1
+ K e5
+ R a2
+ R b3
+ b h5
done
game human human
move b3 b1
move h5 d1
move b1 d1
move e1 f1
move e1 d2
move e1 e2
move e1 f2
move e1 d1
quit
quit
