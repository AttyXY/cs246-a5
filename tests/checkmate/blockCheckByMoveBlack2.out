WELCOME TO CHESS! PLEASE ENTER ONE OF THE FOLLOWING COMMANDS TO BEGIN:
- setup
- game [player1] [player2]

STARTING SETUP

8  --------
7  --------
6  --------
5  --------
4  --------
3  --------
2  --------
1  ----k---
   abcdefgh

8  --------
7  --------
6  --------
5  ----K---
4  --------
3  --------
2  --------
1  ----k---
   abcdefgh

8  --------
7  --------
6  --------
5  ----K---
4  --------
3  --------
2  R-------
1  ----k---
   abcdefgh

8  --------
7  --------
6  --------
5  ----K---
4  --------
3  -R------
2  R-------
1  ----k---
   abcdefgh

8  --------
7  --------
6  --------
5  ----K--b
4  --------
3  -R------
2  R-------
1  ----k---
   abcdefgh

STARTING GAME!

8  --------
7  --------
6  --------
5  ----K--b
4  --------
3  -R------
2  R-------
1  ----k---
   abcdefgh

8  --------
7  --------
6  --------
5  ----K--b
4  --------
3  --------
2  R-------
1  -R--k---
   abcdefgh

Black is in check.
8  --------
7  --------
6  --------
5  ----K---
4  --------
3  --------
2  R-------
1  -R-bk---
   abcdefgh

8  --------
7  --------
6  --------
5  ----K---
4  --------
3  --------
2  R-------
1  ---Rk---
   abcdefgh

Black is in check.
8  --------
7  --------
6  --------
5  ----K---
4  --------
3  --------
2  R-------
1  ---Rk---
   abcdefgh

Black is in check.
Invalid move. Input another move.
8  --------
7  --------
6  --------
5  ----K---
4  --------
3  --------
2  R-------
1  ---Rk---
   abcdefgh

Black is in check.
Invalid move. Input another move.
8  --------
7  --------
6  --------
5  ----K---
4  --------
3  --------
2  R-------
1  ---Rk---
   abcdefgh

Black is in check.
Invalid move. Input another move.
8  --------
7  --------
6  --------
5  ----K---
4  --------
3  --------
2  R-------
1  ---Rk---
   abcdefgh

Black is in check.
Invalid move. Input another move.
8  --------
7  --------
6  --------
5  ----K---
4  --------
3  --------
2  R-------
1  ---k----
   abcdefgh

Final Score:
White: 0
Black: 0
THANKS FOR PLAYING CHESS!
